substrate/client/network/src/config.rs

```diff
@@ -626,6 +629,9 @@ pub struct NetworkConfiguration {
 	/// Maximum number of blocks per request.
 	pub max_blocks_per_request: u32,
 
+	/// Number of peers that need to be connected before warp sync is started.
+	pub min_peers_to_start_warp_sync: usize,
```

Comment:
Please make this an option. Then we don't need to expose the `MIN_PEERS_TO_START_WARP_SYNC` const.